.github/DEVELOPMENT_PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+<!--
+Thanks for submitting a PR to improve our JankariTech blog!
+
+To simplify the review process, please fill in the information below and set the following labels:
+- Assignment: assign to self
+- Reviewers: pick at least one
+-->
+
+## Description
+<!--- Briefly describe your changes -->
+
+## Related Issue
+- Fixes <issue_link>
+- Part of <issue_link>
+
+## Motivation And Context
+<!--- Why is this change required? What problem does it solve? -->
+
+## How Has This Been Tested?
+<!--- Please describe how you tested your changes. -->
+- test environment:
+- test case 1:
+- test case 2:
+
+## Screenshots (If Appropriate):
+
+## Types Of Changes
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to change)
+- [ ] Technical debt
+- [ ] Tests
+- [ ] Documentation
+- [ ] Maintenance (e.g. dependency updates or tooling)
+
+## Checklist:
+- [ ] Code changes
+- [ ] Unit tests added
+- [ ] Acceptance tests added
```

Comment:
Do we have both these levels of automated testing in the blog repo?

Reply:
it seems like so far we don't have any tests our blog... maybe we should add some considering that this is our expertise :)

Shall we keep these points in the template or remove them until we actually do add some tests?

Reply:
I guess that we can keep this, and it will remind us to think about automated tests.

Reply:
opened an issue for this, see https://github.com/JankariTech/blog/issues/131